guide/v3-upgrade.md

```diff
@@ -1,16 +1,16 @@
 # Upgrading to TDS V3
 
-As of **July 1st, 2019** TDS will move feature support from V2: Split Components to V3: Styled Components of the design system. This is being done primarily to facilitate the use of styled-components V4. For a detailed view of our sunsetting schedule, please read our [roadmap page](roadmap.md). Additionally, more information on this subject can be found on the the [Reference Architecture CSS Document](https://github.com/telus/reference-architecture/blob/bb7059d135574c380d2865aa1bbdd633c2345461/development/css.md).
+As of **July 1st, 2019** TDS will move feature support from V2: Split Components to V3: Styled Components of the design system. This is being done primarily to facilitate the use of styled-components V4. For a detailed view of our sunsetting schedule, please read our [roadmap page](roadmap.md). Additionally, more information on this subject can be found on the [Reference Architecture CSS Document](https://github.com/telus/reference-architecture/blob/bb7059d135574c380d2865aa1bbdd633c2345461/development/css.md).
 
 We would also like to note that the following document also applies to Community. Community is now accepting components developed with styled-components, and it is recommended that all new components use it.
 
 ## Requirements
 
-To upgrade to TDS V3, your repo will need **styled-components 4.1** and **React 16.8** installed. These packages are peer dependencies of all TDS V3 components, and lower versions will cause incompatibilities.
+To upgrade to TDS V3, your project will need **styled-components 4.1+** and **React 16.8+** installed. These packages are peer dependencies of all TDS V3 components, and lower versions will cause incompatibilities.
```

Comment:
We should specify that they need to be on styled-components 4.1.x in case someone is looking at then when a new version of styled-components is out. (Unless we're dedicated to bumping our deps when features come out?) Same for react, 16.8.x is more clear.